src/languages/en_US.json

```diff
@@ -3391,12 +3391,16 @@
       "mcap": "MCAP"
     },
     "king_of_hill": {
+      "title": "Launchpad",
       "current_king": "CURRENT KING",
       "last_winner": "LAST WINNER",
       "how_it_works": "How It Works",
       "starting_next_round": "Starting next round",
       "left": "LEFT",
       "king_of_the_hill": "King of the Hill",
+      "name": "Name",
+      "vol": "VOL",
+      "mcap": "MCAP",
```

Comment:
not sure if we want to share these types of labels across different areas of the app or duplicate them like this, i see some others duplicated